src/components/users/UserSelect.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import Select from 'react-select';
+import { FormattedMessage } from 'react-intl';
+
+import './user-form.scss';
+import { black } from 'color-name';
+
+const customStyle = {
```

Comment:
shouldn't this be in a style file? 🤔

Reply:
Are you saying to put this in a scss file?

Reply:
yes